Cargo.toml

```diff
@@ -1,6 +1,11 @@
 [workspace]
 members = [
   "associated-token-account/program",
+  "examples/rust-cross-program-invocation",
```

Comment:
Drop the `rust-` prefix?   ie, default to Rust and then add a `cpp-` prefix for the outliers?

Reply:
There are going to be both Rust and C versions, are we officially making C a second citizen?

Reply:
From an SPL perspective I don't see why not.

Reply:
I'm in favor of keeping both `Rust` and `C` prefix since they align with the docs and it makes it immediately clear from the first glance what language the example is.  We probably won't have other SPL programs written in both languages but the examples we will (and hopefully more in the future like solidity).  Could also do separate directories but that hasn't been a liked approach in the past ;-)

Reply:
Now I'm leaning toward a `c` and `rust` directory for the examples, prefixes are verbose when scaled up in the number of examples and number of languages, plus any language-specific readme can be in the corresponding directory